Sources/BuildServerIntegration/BuildTargetIdentifierExtensions.swift

```diff
@@ -146,7 +146,7 @@ extension BuildTargetIdentifier {
   }
 }
 
-extension BuildTargetIdentifier: CustomLogStringConvertible {
+@_spi(SourceKitLSP) extension BuildTargetIdentifier: @retroactive CustomLogStringConvertible {
```

Comment:
Hmm, should this conformance be in swift-tools-protocols? Seems like most people would need to add it. Though I’m not quite sure which module that conformance would live in.